webui/react/src/utils/set.ts

```diff
@@ -19,15 +19,6 @@ export const union = <T>(setA: Set<T>, setB: Set<T>): Set<T> => {
   return _union;
 };
 
-export const intersection = <T>(setA: Set<T>, setB: Set<T>): Set<T> => {
-  const _intersection = new Set<T>();
-  for (const elem of setB) {
-    if (setA.has(elem)) {
-      _intersection.add(elem);
-    }
-  }
-  return _intersection;
-};
```

Comment:
**nit:** this one seems like it would be useful to keep even if it's not being used now?